baski/server/async_server.py

```diff
@@ -119,7 +137,7 @@ def check_tasks_and_stop(self):
         running_tasks = [t.get_coro() for t in asyncio.all_tasks(loop) if self.should_wait_task(t)]
         if running_tasks:
             logging.warning(f"Wait for tasks complete: {running_tasks}")
-            loop.call_later(1, self.check_tasks_and_stop)
+            loop.call_later(60, self.check_tasks_and_stop)
```

Comment:
Let's check more frequently. The server may have stopped in a matter of seconds. We don't need to wait whole minute. 

I should not be any harm to `call self.check_tasks_and_stop` frequently